### PR TITLE
Events management and bugfixes

### PR DESCRIPTION
**Pull request by ThePletch**

### CODE REVIEW

File: app/views/welcome/index.html.slim [hunk 1]
```diff
@@ -1,26 +1,34 @@
-div.container-fluid
```
Comment (ThePletch) [inline app/views/welcome/index.html.slim]: whoops, meant to delete this entirely

**Comment by Maelstroms** [comment]
For the welcome spec, do we not want to keep it for the purpose of test completion when Mike looks over our code at the end of the class?

**Comment by ThePletch** [comment]
The welcome page was a placeholder and I'm deleting it from existence entirely.

**Comment by jonathanchiu** [comment]
Is there a way to create an event (i.e. a `link_to`)? Or is that not considered for now as part of this PR?

**Comment by ThePletch** [comment]
Event creation is done through the admin panel (and intentionally restricted to there).

**Comment by jonathanchiu** [comment]
Nice, everything looks very solid +1

Once we get this merged along with my header and footer PR, I think things will start looking really polished :clap:

**Pull request merged**